config/default-spinnaker-local.yml

```diff
@@ -121,6 +121,21 @@ providers:
       # A path to a plain text file containing the user's password
       passwordFile: ${SPINNAKER_DOCKER_PASSWORD_FILE}
 
+  openstack:
+    # For more information on configuring Openstack clusters, see
+    # TODO add link to docs when available
+
+    enabled: ${SPINNAKER_OPENSTACK_ENABLED:false}
+    primaryCredentials:
+      name: my-openstack-account
+      authUrl:
+      username:
+      password:
+      projectName:
+      domainName:
```

Comment:
This is overriding the defaults in spinnaker.yml with the empty string. And all the others are also overriding the environment variables in the spinnaker.yml with empty. I think you either want nothing here or to copy what you have in spinnaker.yml.

For now copying would probably be consistent with all the other platforms, but I'd like to get rid of this file or at least only put the absolute minimal required things here because they have no meaningful default values.